tests/wpt/mozilla/tests/mozilla/remove_link_styles.html

```diff
@@ -0,0 +1,19 @@
+<!doctype html>
+<meta charset="utf-8">
+<title>Removing link tag should remove associated styles</title>
+<link rel="stylesheet" type="text/css" href="remove_link_styles.css">
+<link rel="match" href="remove_link_styles_ref.html">
+<body>
+    <style>
+        body {
+            color: green;
+        }
+    </style>
+
+    This text should be green and the background should not be red.
+
+    <script>
+        var l = document.querySelector('link[rel="stylesheet"]');
+        l.parentNode.removeChild(l);
```

Comment:
The `color` part of this tests is useless because the second `<style>` element will override it no matter what.

Could you instead use `color: white !important`?